### DISCUSSION TITLE
https://nolebase-template-iota.vercel.app/%E7%AC%94%E8%AE%B0/

### DISCUSSION THREAD

**Discussion by giscus** [Jackiexiao/nolebase-template#2] [Announcements]
# https://nolebase-template-iota.vercel.app/%E7%AC%94%E8%AE%B0/

记录回忆，知识和畅想的地方

https://nolebase-template-iota.vercel.app/%E7%AC%94%E8%AE%B0/

**Comment by Everett406** [comment]
hi
